girder/utility/s3_assetstore_adapter.py

```diff
@@ -165,12 +165,12 @@ def __init__(self, assetstore):
         """
         :param assetstore: The assetstore to act on.
         """
+        super(S3AssetstoreAdapter, self).__init__(assetstore)
         if ('accessKeyId' in assetstore and 'secret' in assetstore and
                 'service' in assetstore):
             assetstore['botoConnect'] = makeBotoConnectParams(
                 assetstore['accessKeyId'], assetstore['secret'],
                 assetstore['service'])
-        self.assetstore = assetstore
```

Comment:
Good point, but I'd say just change all the `assetstore[...] ` to use `self.assetstore[...]`. It's idiomatic that the superclass constructor be called first, unless you need to do some special setup.